docs/configuration/configuration.md

```diff
@@ -1603,7 +1603,7 @@ Available meta labels:
   from underlying pods), the following labels are attached:
 * `__meta_kubernetes_endpointslice_address_target_kind`: Kind of the referenced object.
 * `__meta_kubernetes_endpointslice_address_target_name`: Name of referenced object.
-* `__meta_kubernetes_endpointslice_address_type`: The ip protocol family of the adress target.
+* `__meta_kubernetes_endpointslice_address_type`: The ip protocol family of the address target.
```

Comment:
I believe this description intends to indicate the address possessed by the target, so we can make it more clear by fixing the grammar.

```suggestion
* `__meta_kubernetes_endpointslice_address_type`: The ip protocol family of the address of the target.
```

Reply:
Thanks! Done. PTAL.